pkg/test/framework/components/echo/kube/flags.go

```diff
@@ -0,0 +1,35 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package kube
+
+import "flag"
+
+var (
+	serviceTemplateFile      = "service.yaml"
+	deploymentTemplateFile   = "deployment.yaml"
+	vmDeploymentTemplateFile = "vm_deployment.yaml"
+)
+
+func init() {
+	flag.StringVar(&serviceTemplateFile, "istio.test.echo.kube.template.service", serviceTemplateFile,
```

Comment:
nit: indicate that a default will be selected if not specified. Same comment applies to all.